tests/GasModelSpec.hs

```diff
@@ -114,9 +115,13 @@ runTest :: GasUnitTests -> IO [GasTestResult ([Term Name], EvalState)]
 runTest t = mapOverGasUnitTests t run run
   where
     run expr dbSetup = do
-      (res, st) <- bracket (setupEnv dbSetup) (gasSetupCleanup dbSetup) (mockRun expr)
+      (res, st) <- bracket (setupEnv' dbSetup) (gasSetupCleanup dbSetup) (mockRun expr)
       res' <- eitherDie (getDescription expr dbSetup) res
       return (res', st)
+    setupEnv' dbs = do
+      (r, s) <- setupEnv dbs
+      let r' = set eeExecutionConfig (mkExecutionConfig [FlagDisableInlineMemCheck]) r
```

Comment:
why do we need to disable?

Reply:
Gasmodelspec is parsing a module :(

Reply:
I was also wondering the same thing re: disabling in mem check here.
@jmcardon can you elaborate on what you mean by

> Gasmodelspec is parsing a module :(

Is it that the expected values have change as a result of enabling in-memory check?